src/config-import.js

```diff
@@ -330,11 +330,14 @@ function translationsForLanguage(warnings) {
    *
    */
   return function* (lang, languageTranslations) {
-    const { language: languageCode, region: regionCode } = parseBCP47(lang)
+    const parsed = parseBCP47(lang)
+    const { language: languageCode } = parsed
     if (!languageCode) {
       warnings.push(new Error(`invalid translation language ${lang}`))
       return
     }
+    let { region: regionCode } = parsed
+    regionCode ||= undefined
```

Comment:
nit: I assume this is to catch regionCode being `null`. This syntax is new/obscure enough that I think this warrants a comment, or changing to code to something more explicit like `if (regionCode === null) regionCode = undefined`

Reply:
It's to catch `regionCode` being falsy. `''`, `null`, and `undefined` should all be coerced to `null`.

This is because [`regionCode` has to be between 2 and 3 characters when defined](https://github.com/digidem/comapeo-schema/blob/74a3d0da01282e664653924533bab836f933961f/schema/translation/v1.json#L54-L59); the empty string is not valid.